### ISSUE TITLE
NTR: Abdominal imaging: retroperitoneal mass (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9012]
New term request Abdominal imaging: retroperitoneal mass (nadea3_181121015809): Includes extragonadal germ cell tumor  UMLS: 0267771

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000285
- label: Retroperitoneal mass
- parents: [Abnormality of the abdominal organs (HP:0002012)](https://hpo.jax.org/app/browse/term/HP:0002012)
- pmid: 
- def: A swelling or enlargment localized to the retroperitoneal region. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9012
- synonyms:
